DigitalLearningSolutions.Web/Styles/shared/searchableElements/sort.scss

```diff
@@ -1,99 +1,81 @@
 @import "~nhsuk-frontend/packages/core/all";
 @import "common";
 
-.sort-dropdown {
-  border-radius: 4px;
+.sort-form {
+  width: 100%;
 }
 
-.sort-by-dropdown {
-  @extend .left-dropdown;
-}
-
-.sort-direction-dropdown {
-  @extend .input-with-no-js-submit-button;
+.sort-box-flex-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 100%;
 
-  width: 63%;
+  .sort-by-dropdown {
+    @extend .left-dropdown;
+    display: flex;
 
-  @include mq($until: desktop) {
-    .js-enabled & {
-      width: 70%;
+    @include mq($until: tablet) {
+      width: 100%;
     }
-  }
-
-  @include mq($from: $xl-desktop) {
-    width: 74%;
-  }
-}
 
-.sort-direction-dropdown-container {
-  @include mq($from: desktop) {
-    width: calc(59% - 12px);
-    float: left;
+    @include mq($from: tablet) {
+      flex: 1;
+    }
   }
-}
 
-.sort-submit {
-  @extend .right-submit-button;
-}
+  .sort-order-button-flex-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
 
-.sort-row {
-  display: flex;
-  flex-direction: row;
+    .search-sort-flex-container & {
+      @include mq($from: tablet) {
+        width: 50%;
+      }
+    }
 
-  @include mq($until: desktop) {
-    margin-top: 36px;
-    margin-left: 16px;
-    margin-right: 16px;
-    flex-direction: column;
+    @include mq($from: tablet) {
+      width: 60%;
+    }
 
-    .js-enabled & {
-      flex-direction: row;
+    @include mq($from: large-desktop) {
+      .js-enabled & {
+        width: 33.3333%;
+      }
     }
-  }
 
-  @include mq($until: 339px) {
-    flex-direction: column !important;
-  }
-}
+    .sort-direction-dropdown {
+      @extend .input-with-no-js-submit-button;
+      display: flex;
 
-.sort-select-container {
-  padding-right: 8px;
-  padding-top: 8px;
+      @include mq($until: tablet) {
+        width: 100%;
+      }
 
-  @include mq($from: desktop) {
-    padding-left: 8px;
-    margin-top: -8px;
-  }
-}
+      @include mq($from: tablet) {
+        flex: 1;
+      }
 
-.sort-row::before {
-  content: "Sort by:";
-  position: relative;
-  top: -32px;
-  height: 0;
-  width: 0;
-  float: left;
-  font-weight: 600;
-  font-size: 19px;
-  color: rgb(33, 43, 50);
-  font-family: "Frutiger W01", Arial, sans-serif;
-  white-space: nowrap;
-
-  @include mq($from: desktop) {
-    left: 8px;
-  }
-}
+      .js-enabled & {
+        width: 100%;
+      }
+    }
 
-.sort-row .nhsuk-button {
-  margin-top: 0;
-  margin-bottom: 0;
-}
+    .sort-submit {
+      @extend .right-submit-button;
+      background-color: $button-dark-blue;
+      min-width: 100px;

```

Comment:
Where is the min-width coming from?

Reply:
Ah whoops, it shouldn't be defined here too, it's on the right-submit-button too. Essentially it's the size so that the buttons don't get unnecessarily squashed at mobile layouts and we don't have to set more percentages to make it work.

![image](https://user-images.githubusercontent.com/59561751/133422537-50001e72-418a-4f9b-ab41-2ccdf929f66f.png)